releases/cockroachcloud-11082021.md

```diff
@@ -0,0 +1,37 @@
+---
+title: What's New in CockroachDB Cloud
+toc: true
+summary: Additions and changes in CockroachDB Cloud since October 3, 2021 .
+---
+
+## November 8, 2021
+
+Get future release notes emailed to you:
+
+{% include marketo.html %}
+
+### General changes
+
+- {{ site.data.products.serverless }}, a fully-managed, auto-scaling deployment of CockroachDB, is now available. To get started with {{ site.data.products.serverless }} for free, see the [Quickstart](../cockroachcloud/quickstart.html).
+- CockroachCloud Free (beta) and CockroachCloud are now {{ site.data.products.serverless }} and {{ site.data.products.dedicated }}, respectively. Your ability to use your cluster will not be affected.
+- {{ site.data.products.serverless }} clusters will now run CockroachDB [v21.2.0-beta.4](v21.2.0-beta.4.html).
+- New {{ site.data.products.dedicated }} clusters will now run CockroachDB [v21.1.11](v21.1.11.html).
+
+### Console changes
```

Comment:
Hi @gemma-shay,

I think we need something along the lines of (feel free to update): 

CockroachDB Serverless now supports monitoring workload activity by introducing Sessions, Transactions, and Statements pages under the SQL Activity navigation menu. Statements and Transaction statistics are also persisted on disc allowing longer retention for monitoring activity and performance trends over time.

I'm following up with the team why this was not captured via the release notes generator script.